recipes/libdai/recipe/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles"  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
```

Comment:
Is there a `nmake check` that we can run?

Reply:
I can add a  `make check` if appreciated once I am back from vacations